tests/test_constantpropagation.py

```diff
@@ -9,7 +9,7 @@
 
 def test_libc_x86():
 
-    p = angr.Project(os.path.join(test_location, "i386", "libc-2.27-3ubuntu1.so.6"), auto_load_libs=True)
+    p = angr.Project(os.path.join(test_location, "i386", "libc-2.27-3ubuntu1.so.6"), auto_load_libs=True)#disabling auto_load_libs increases the execution time.
```

Comment:
I think the comment would be better on the line above this. Also please have whitespace around the `#`.

Reply:
Changes done.

Reply:
They do not appear to be